app/views/categories/index.html.erb

```diff
@@ -0,0 +1,76 @@
+<div class="container">
+  <h1 class="categories-header">Categories List</h1>
+  <div class="row">
+    <% @categories.each do |category| %>
+      <div class="col-xs-4">
+        <div class="panel panel-primary">
+
+          <div class="header-location panel-heading">
+            <h2>
+              <%= category.name %>
+              <button type="button" class="btn btn-default btn-sm categories-search-button" data-toggle="modal" data-target="#myModal-<%=category.id %>">
+                <span class="glyphicon glyphicon-search"></span>
+              </button>
+            </h2>
+
+            <div class="modal fade" id="myModal-<%=category.id %>" tabindex="-1" role="dialog" aria-labelledby="myModalLabel">
+              <div class="modal-dialog" role="document">
+                <div class="modal-content">
+                  <div class="modal-header bg-primary">
+                    <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+                    <h4 class="modal-title" id="myModalLabel"><%=category.name %> Search</h4>
+                  </div>
+                  <form class="navbar-form" method="GET" action="/search">
+                      <input type="hidden" name="cat_id" value="<%= category.id %>">
+                      <input type="text" name="search" class="form-control" placeholder="Search">
+                    <button type="submit" class="btn btn-default">Submit</button>
+                  </form>
+                  <div class="modal-footer">
+                    <button type="button" class="btn btn-default" data-dismiss="modal">Close</button>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+
+
+
+          <div class="panel-body">
+            <h4>Top Picks</h4>
+            <ul class="list-group">
+              <% category.top_five_goals.each do |goal| %>
+                <li class="row list-group-item">
+                  <div class="col-md-10">
+
+                    <a tabindex="0" data-html="true" role="button" data-trigger="focus" data-toggle="popover" data-placement="auto" title='<a href="/goals/<%= goal.id %>"><%= goal.task %></a>' data-content="
+                      <% if !!goal.data %>
```

Comment:
This should just be truthy. You only need to do the `!!` if you need a true boolean